spec/queries/mentoring_query_spec.rb

```diff
@@ -6,25 +6,74 @@
   describe '#call' do
     subject(:call) { described_class.new.call }
 
-    context 'when there are mentees and mentors' do
+    context 'when there are one mentee and mentor' do
       let(:mentor) { create(:user) }
       let!(:mentee) { create(:user, reviewer_id: mentor.id) }
 
-      let(:mentors) { call.map(&:mentors) }
+      let(:mentors) { call.map(&:name) }
       let(:offices) { call.map(&:city) }
       let(:mentees) { call.map(&:mentees) }
 
-      it 'returns mentors' do
+      it 'returns mentor' do
         expect(mentors).to contain_exactly(mentor.name)
       end
 
       it "returns mentor's office" do
         expect(offices).to contain_exactly(mentor.office.city)
       end
 
-      it "returns mentor's mentees" do
+      it "returns mentor's mentee" do
         expect(mentees).to contain_exactly([mentee.name])
       end
     end
+    
+    context 'when there are multiple mentees and mentors' do
+      let(:bob) { create(:user) }
+      let!(:bob_mentee) { create(:user, reviewer_id: bob.id) }
+      let!(:other_bob_mentee) { create(:user, reviewer_id: bob.id) }
+      
+      let(:brown) { create(:user) }
+      let!(:brown_mentee) { create(:user, reviewer_id: brown.id) }
+      let!(:other_brown_mentee) { create(:user, reviewer_id: brown.id) }
+      
+      let(:mentors) { call.map(&:name) }
+      let(:offices) { call.map(&:city) }
+      let(:mentees) { call.map(&:mentees) }
+
+      it 'returns mentors' do
+        expect(mentors).to contain_exactly(bob.name, brown.name)
+      end
+
+      it "returns mentors's offices" do
+        expect(offices).to contain_exactly(bob.office.city, brown.office.city)
+      end
+
+      it "returns mentor's mentees" do
+        expect(mentees.flatten).to contain_exactly(bob_mentee.name, other_bob_mentee.name, brown_mentee.name, other_brown_mentee.name)
+      end
+    end
+    
+    context 'when there are no mentees or mentor' do
+      let!(:bob) { create(:user) }
+      let!(:brow) { create(:user) }
+      let!(:john) { create(:user) }
+      let(:joe) { create(:user) }
```

Comment:
since we're not going to use any of these values separately, we can create them and not assign to any variable:
```suggestion
      before do
        4.times { create(:user) }
      end
```

Reply:
> Also noticed there were some trailing whitespaces on the spec file, so watch out for that eyes

I'll pass rubocop one more time in those files.